modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGoCodegen.java

```diff
@@ -391,7 +397,7 @@ public String getSchemaType(Schema p) {
             type = openAPIType;
         } else if ("object".equals(openAPIType) && ModelUtils.isAnyTypeSchema(p)) {
             // Arbitrary type. Note this is not the same thing as free-form object.
```

Comment:
We should remove/revise this comment (not added by you) as it's not entirely true.